html/pfappserver/lib/pfappserver/I18N/fr.po

```diff
@@ -12402,6 +12402,65 @@ msgid ""
 "Maintenance section)"
 msgstr "Archive les adresses IPs des appareils sur le réseau quand qu'activé (voir la section Maintenance)"
 
+# pf::action (SYSLOG_PARSER_REGEX_VIOLATION_ACTIONS)
```

Comment:
You shouldn't be touching the fr.po since its handled by transifex, just revert it back to whatever it was and its good for merging